### PR TITLE
Bug 1742448 - Allow passing in the build date

### PR DESCRIPTION
**Pull request by badboy**
Requires https://github.com/mozilla/glean_parser/pull/431 & a release.
Requires https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/715

### CODE REVIEW

**Comment by badboy** [comment]
Adding multiple reviewers, whoever gets here first.
I'd like the review first before I land the corresponding code.

File: samples/ios/app/Cartfile
Comment (travis79) [inline samples/ios/app/Cartfile:3]: Personal repo use again

**Review by travis79 — APPROVED** [review]
This all looks good, with just a couple of comments about things pointing to your personal GitHub. No reason to block on this as long as we have a followup bug to fix it.

File: glean-core/ios/sdk_generator.sh
Comment (badboy) [reply glean-core/ios/sdk_generator.sh:156]: yes, as said I want a code review here before I land the corresponding code, because if anything changes it might also require glean_parser changes ,etc.

Comment (travis79) [reply glean-core/ios/sdk_generator.sh:156]: Got it, makes sense

**Comment by badboy** [comment]
Drop in coverage is expected. We only record coverage from Rust tests and currently the build date is not used from within Rust

**Pull request merged**
